build/tutorials/tutorials-contest/pangolin-token-subgraph/README.md

```diff
@@ -0,0 +1,480 @@
+
+
+Source: [https://thegraph.com/legacy-explorer/subgraph/dasconnor/pangolin-dex?selected=playground](https://thegraph.com/legacy-explorer/subgraph/dasconnor/pangolin-dex?selected=playground)
+
+The playground consists of the 3 sections, the query section, the results section, and the schema section. You can modify queries, click on the play button and have the returned data displayed in the middle section (results section). The schema section enables you to go through the entities and their associated fields to know what kind of queries are supported by that particular subgraph. Below is an example query you can initiate on the Pangolin exchange subgraph.
+
+
+```graphql
+{
+  tokens(first: 2) {
+    id
+    symbol
+    name
+    decimals
+  }
+}
+```
+
+The query above is asking for the first 2 tokens from the results of the Pangolin exchange subgraph and it is interested in the fields - `id`, `symbol`, `name`, and `decimals`. The returned result is shown below.
+
+```graphql
+{
+  "data": {
+    "tokens": [
+      {
+        "decimals": "18",
+        "id": "0x008e26068b3eb40b443d3ea88c1ff99b789c10f7",
+        "name": "Zero.Exchange Token",
+        "symbol": "ZERO"
+      },
+      {
+        "decimals": "18",
+        "id": "0x020ef96c76a225fc0151c191b2a15accc915b68c",
+        "name": "WOUF",
+        "symbol": "WOUF"
+      }
+    ]
+  }
+}
+```
+
+Using the schema, you can browse through the Pangolin API to discover other entities you are interested in. For example, you can construct a query to know the daily volume on Pangolin and the number of transactions carried out on the [DEX](https://support.avax.network/en/articles/4587136-what-is-a-decentralized-exchange-dex).
+
+```graphql
+{
+  pangolinDayDatas(first: 3, orderBy: date) {
+    date
+    txCount
+    dailyVolumeETH
+    dailyVolumeUSD
+  }
+}
+```
+
+The query above will request the data from the first 3 days and only the fields specified will be returned.
+
+```graphql
+{
+  "data": {
+    "pangolinDayDatas": [
+      {
+        "dailyVolumeETH": "0",
+        "dailyVolumeUSD": "0",
+        "date": 1612742400,
+        "txCount": "2"
+      },
+      {
+        "dailyVolumeETH": "52995.55027820444446352341877709734",
+        "dailyVolumeUSD": "1497153.144890008456466344638128839",
+        "date": 1612828800,
+        "txCount": "3603"
+      },
+      {
+        "dailyVolumeETH": "381001.9576775585402832922123431514",
+        "dailyVolumeUSD": "15686187.55247135303384368066876295",
+        "date": 1612915200,
+        "txCount": "12447"
+      }
+    ]
+  }
+}
+```
+
+The result shows that there was no volume on the first day but the volume and the number of transactions increased steadily from the second day to the third day.
+
+More data can be gotten from this particular entity by constructing queries that match the fields supported. These fields can be identified from the schema as shown in the image below.
+
+
+![alt_text](images/image4.png "Pangolin DEX Schema")
```

Comment:
Rename images and paths

Reply:
Images names have been changed to the naming convention.